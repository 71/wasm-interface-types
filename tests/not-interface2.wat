;; parse-fail

(module

  (@interface foo))

(; CHECK-ALL:
unexpected token, expected one of: `type`, `import`, `export`, `func`, `implement`
     --> tests/not-interface2.wat:5:15
      |
    5 |   (@interface foo))
      |               ^
;)
