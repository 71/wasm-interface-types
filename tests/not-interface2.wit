;; parse-fail

(module

  (@interface foo))
