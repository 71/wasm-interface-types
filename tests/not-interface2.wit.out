unexpected token, expected one of: `type`, `import`, `export`, `func`, `implement`
     --> ../../tests/not-interface2.wit:5:15
      |
    5 |   (@interface foo))
      |               ^
